### PR TITLE
London 10 Paulina and Andrius

### PR DESCRIPTION
**Pull request by PaulinaWywrot**

### CODE REVIEW

File: questions-js1/1.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Q: How do you write "Hello World" to the console?
 
 A: {YOUR ANSWER HERE}
-
+console.log("Hello World");
```
Comment (AnnaFYZ) [inline questions-js1/1.md:4]: well done

File: questions-js1/3.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 Q: What is the difference between `let` and `const`?
 
 A: {YOUR ANSWER HERE}
-
+you should use const when you don't want to change the value in the future
+you should use let if the value of your variable will change in the future
```
Comment (AnnaFYZ) [inline questions-js1/3.md:5]: well done

**Review by AnnaFYZ — COMMENTED** [review]
thank you

File: questions-js1/3.md (hunk 2 above)
Comment (Junitalama) [reply questions-js1/3.md:5]: good job paulina

File: questions-js1/1.md (hunk 1 above)
Comment (PaulinaWywrot) [reply questions-js1/1.md:4]: thank you

**Pull request closed without merging**